inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java

```diff
@@ -111,12 +112,25 @@ public InlongConsumeInfo getFromEntity(InlongConsumeEntity entity) {
         }
         String groupId = entity.getInlongGroupId();
         InlongGroupInfo groupInfo = groupService.get(groupId);
+        if (!(groupInfo instanceof InlongPulsarInfo)) {
```

Comment:
Why not use mqType to convert directly?

Reply:
It's just a double-check, which seems unnecessary and has been removed.